### PR TITLE
Better client logs

### PR DESCRIPTION
**Pull request by YLivay**

### CODE REVIEW

File: src/app/components/client/system-report-modal/system-report-form.vue [hunk 1]
```diff
@@ -1,5 +1,24 @@
 <template>
 	<app-form name="systemReportForm">
+		<app-form-group name="issue_tracker_url" optional :label="$gettext('Issue Tracker Link')">
+			<p class="help-block">
+				<translate>
+					Paste the link to your ticket here. It's optional, but without it we can't get back to you
+					if we need more info to solve it.
+				</translate>
+				<span class="emoji emoji-huh" />
```
Comment (nilllzz) [inline src/app/components/client/system-report-modal/system-report-form.vue]: Eh? No app-jolticon?

File: src/app/components/client/system-report-modal/system-report-form.vue [hunk 2]
```diff
@@ -1,5 +1,24 @@
 <template>
 	<app-form name="systemReportForm">
+		<app-form-group name="issue_tracker_url" optional :label="$gettext('Issue Tracker Link')">
+			<p class="help-block">
+				<translate>
+					Paste the link to your ticket here. It's optional, but without it we can't get back to you
```
Comment (nilllzz) [inline src/app/components/client/system-report-modal/system-report-form.vue]: I think we should directly link to the issue tracker project here.

File: src/app/components/client/system-report-modal/system-report-form.vue [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+					if we need more info to solve it.
+				</translate>
+				<span class="emoji emoji-huh" />
+			</p>
+
+			<app-form-control
+				type="text"
+				placeholder="https://github.com/gamejolt/issue-tracker/issues/1313"
+				:rules="{
+					max: 255,
```
Comment (nilllzz) [inline src/app/components/client/system-report-modal/system-report-form.vue]: Probably should also have a min length (of 42, which would be the length of this: `github.com/gamejolt/issue-tracker/issues/`), or a regex to test for that url shape.

File: src/app/components/client/system-report-modal/system-report-form.vue (hunk 1 above)
Comment (YLivay) [reply src/app/components/client/system-report-modal/system-report-form.vue]: It's a game jolt emoji, not a jolticon. iirc we don't have our emojis as jolticons.

File: src/app/components/client/system-report-modal/system-report-form.vue (hunk 2 above)
Comment (YLivay) [reply src/app/components/client/system-report-modal/system-report-form.vue]: We do in the modal that uses the form so its clear enough. It's also the only link there so it stands out too.

**Pull request merged**